next/components/forms/segments/AccountSections/ThankYouSection/ThankYouSection.tsx

```diff
@@ -44,6 +45,8 @@ const ThankYouSection = () => {
             content={t(`thank_you.result.${status}.content`)}
             firstButtonTitle={t('thank_you.button_to_formular_text')}
             secondButtonTitle={t('thank_you.button_to_profil_text')}
+            feedbackTitle={t(`thank_you.result.${status}.feedback_title`)}
```

Comment:
I think it's is not good to have translations like this. It gets broken every time and you can never find the reference when searching for usages. Please create map like:
```
const map = {
'status1': 'thank_you.result.status1.feedback_title',
'status2': 'thank_you.result.status2.feedback_title'
}
```
And then use it, thanks.

Reply:
Also did it for title and content